app/views/declaration/transport_container_summary.scala.html

```diff
@@ -64,11 +64,11 @@
                         attributes = Map("id" -> s"containers-row$index-seals")
                     ),
                     TableRow(
-                        content = HtmlContent(changeLink(messages("site.remove"), messages("declaration.transportInformation.containers.remove.hint", container.id), TransportContainerController.submitContainerRemove(mode, container.id))),
+                        content = HtmlContent(changeLink(messages("site.remove"), Some(messages("declaration.transportInformation.containers.remove.hint", container.id)), TransportContainerController.submitContainerRemove(mode, container.id))),
```

Comment:
Good idea "changing" the changeLink to Link.   I realised after I merged that I was using it to "remove"